tests/run_all

```diff
@@ -845,12 +834,6 @@ run_test ./usertools/usermod/49_usermod_change_gid+move_homedir_other_device/use
 run_test ./usertools/usermod/50_usermod_change_uid+move_homedir/usermod.test
 run_test ./usertools/usermod/51_usermod_change_gid+move_homedir/usermod.test
 run_test ./usertools/usermod/52_usermod_move_homedir_symlink/usermod.test
-run_test ./cptools/01/run1
-run_test ./cptools/01/run2
-run_test ./cptools/01/run3
-run_test ./cptools/01/run4
-run_test ./cktools/01/run1
-run_test ./cktools/01/run2
```

Comment:
cktools tests still exist, no?

Reply:
Sigh, yes. Shouldn't do this in the middle of 5 other things.

Reply:
:)